src/modules/InstShowInfo.rb

```diff
@@ -13,10 +14,24 @@ def main
 
       Yast.import "Report"
       Yast.import "Label"
+
+      @shown_info_files = []
     end
 
-    # @param [String] info_file path to be shown
-    def show_info_txt(info_file)
+    # @param file_path [String] path to be shown
+    def show_info_txt(file_path)
+      info_file = Y2Packager::InfoFile.read(file_path)
+      if info_file.nil?
+        Builtins.y2milestone("No %1", info_file)
```

Comment:
:thinking: this will log "No nil", isn't it?

Probably a _leftover_. Maybe you wanted use _file_path_ instead

Reply:
Yes, a leftover.